src/sql/workbench/contrib/notebook/browser/markdownToolbarActions.ts

```diff
@@ -150,6 +161,33 @@ export class MarkdownTextTransformer {
 		}
 	}
 
+	/**
+	 * Instantiate modal for use as callout when inserting Link or Image into markdown.
+	 * @param calloutStyle Style of callout passed in to determine which callout is rendered
+	 */
+	private async createCallout(calloutStyle: CalloutStyle, triggerElement: HTMLElement) {
+		const posX = triggerElement.getBoundingClientRect().left;
+		const posY = triggerElement.getBoundingClientRect().top;
+		let title = calloutStyle.toString().toLowerCase();
+
+		if (title.includes('image')) {
+			title = this.insertImageHeading;
+		} else {
+			title = this.insertLinkHeading;
+		}
+
+		if (!this._callout) {
+			const dialogXYOffset = { xOffset: 22, yOffset: 24 };
```

Comment:
Comment explaining why you chose these values would be great

Reply:
Comment added. It's for Mac clients.